### ISSUE TITLE
Escaped new lines after keywords not supported

### ISSUE THREAD

**Issue by jonathan-jaekel** [RedCoolBeans/dockerlint#14]
### Steps to reproduce/test case

``` bash
$ cat > Dockerfile << 'EOF'
FROM centos

CMD \
  ["bash"]
EOF

$ dockerlint Dockerfile
check: from_first
check: no_empty_tag
check: no_empty_digest
check: json_array_brackets
check: json_array_even_quotes
check: json_array_format
DEBUG:'Arg: ["bash"]'
check: recommended_exec_form
WARN:  Recommended exec/array form not used on line 3
check: add
check: multiple_entries
check: sudo
check: absolute_workdir
check: onbuild_copyadd
check: onbuild_disallowed
check: label_no_empty_value

ERROR: Dockerfile failed.
```
### Affected version(s)

Dockerlint 0.2.0
